packages/core/src/components/document-component/document-component.tsx

```diff
@@ -310,22 +342,24 @@ export class DocumentComponent implements ComponentInterface {
                   <dso-icon icon={this.recursiveToggle === true ? "eye" : "eye-slash"} />
                 </button>
               )}
-              {this.genesteOntwerpInformatie && !this.open && !this.bevatOntwerpInformatie && (
-                <>
-                  <dso-badge status="warning" aria-describedby="nested-draft-description">
-                    !
-                  </dso-badge>
-                  <dso-tooltip id="nested-draft-description">Er is een ontwerp beschikbaar.</dso-tooltip>
-                </>
-              )}
+              {this.genesteOntwerpInformatie &&
+                !this.bevatOntwerpInformatie &&
+                ((!this.open && this.mode === "document") || this.mode === "table-of-contents") && (
```

Comment:
Mijn hersens krijgen dit niet verwerkt, kan je dit toelichten?

Reply:
Ik doe een poging.

Een badge wordt getoond achter de Heading indien dit Document Component niet in Ontwerp is en wel kindjes heeft die in Ontwerp zijn en in de document-modus allen als het Document Component niet open is, terwijl in de table-of-contents modus open er niet toe doet.

Reply:
Ik begrijp het nu beter, maar we moeten dit beter borgen voor toekomstige generaties.

Zullen we deze test onderbrengen in een instance method?

```ts
showOntwerpLabel(): boolean {
  // leesbare code
}
```

```ts
{this.showOntwerpLabel() && (
  <>...</>
)}
```

Reply:
Ik heb de method `showOntwerpBadge` genoemd.